### PR TITLE
Bjorn/Crown support (rudimentary)

### PR DESCRIPTION
**Pull request by horrible-little-slime**

### CODE REVIEW

**Comment by horrible-little-slime** [comment]
@gausie I plan to add bjorn-picking support after this, but I'd like to get this in before I start doing fancy stuff

@phulin I'm 80% sure you'll be fine with the names I have chosen here

File: src/modifier.ts [hunk 1]
```diff
@@ -97,3 +97,51 @@ export function get(
     return statModifier(subject as Effect, name);
   }
 }
+
+export type BooleanModifierValues = Partial<
```
Comment (gausie) [inline src/modifier.ts:101]: Is there a benefit to having them split via type here? could we not just have

```ts
export type Modifiers = Partial<{
  [modifier in BooleanModifier]: boolean;
  [modifier in ClassModifier]: Class;
  ...
}>;
```

**Comment by horrible-little-slime** [comment]
I think there is likely benefit to suggesting specificity, but mostly I got
red squiggles on my first try

File: src/modifier.ts (hunk 1 above)
Comment (horrible-little-slime) [reply src/modifier.ts:101]: I think there is likely benefit to suggesting specificity, but mostly I got
red squiggles on my first try

Comment (horrible-little-slime) [reply src/modifier.ts:101]: I can set it up like properties, though

**Review by gausie — APPROVED** [review]

**Pull request merged**